packages/flutter/lib/src/material/material.dart

```diff
@@ -114,9 +114,11 @@ abstract class MaterialInkController {
 ///
```

Comment:
maybe rephrase the last paragraph above the Shape header to say that elevation and shadowColor are animated, and not the others.

Reply:
done